gsa/src/web/entities/useEntitiesTimeout.js

```diff
@@ -0,0 +1,39 @@
+/* Copyright (C) 2020 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation, either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {useCallback} from 'react';
+import {isDefined} from 'gmp/utils/identity';
+
+const useEntitiesTimeout = ({entities, gmpSettings}) => {
```

Comment:
Sorry for being so late. Any reason for not using `useGmpSettings` hook here directly instead of passing the setting?

Reply:
Personally I find the name a bit confusing because we have useSessionTimeout which returns some functions to handle the timing and this hook just calculates the duration for this timing. It just confused my mind when reading the code and I don't know a better name at the moment.

Reply:
> Sorry for being so late. Any reason for not using useGmpSettings hook here directly instead of passing the setting?

Nope, no reason. Just didn't think of it because I didn't notice until you pointed it out that gmpSettings is only used by this function and nowhere else as I just focused on translating the page at first.